modules/doc/content/application_usage/input_syntax.md

```diff
@@ -134,71 +134,27 @@ can include other files. The only requirement is that the included files
 must contain a set of syntactically complete blocks or parameters.
 
 Functionally, including a file is equivalent to inserting the
-text of the file at the `!include` location. Therefore,
-for example, if the included file uses a value `${somevar}` defined in the parent file
-via `somevar = somval`, then the `!include` statement must occur *after* the
-`somevar` definition.
-
-!alert! warning title=Beware merge-related include ordering issues
-When an included file contains the same block as the parent file, the blocks get
-*merged*. The merge moves the second instance of the block into the first, not
-the first into the second. For example,
-
-`file1.i`:
-
-```
-[BlockA]
-  param1 = 4
-[]
-
-!include file2.i
-```
-
-`file2.i`:
-
-```
-val3 = 8
-
-[BlockA]
-  param2 = ${val3}
-[]
-```
-
-This effectively produces the input
-
-```
-[BlockA]
-  param1 = 4
-[]
-
-val3 = 8
-
-[BlockA]
-  param2 = ${val3}
-[]
-```
-
-and after the merge operation,
-
-```
-[BlockA]
-  param1 = 4
-  param2 = ${val3}
-[]
-
-val3 = 8
-```
-
-This results in an error, since `${val3}` is now used before it is defined.
-Therefore, files must be partitioned with this behavior in mind.
-!alert-end!
+text of the file at the `!include` location. If a block already exists, then
+its sub-blocks and parameters are merged into the existing block to avoid
+having multiple blocks of the same name. Note, however, that this merging
+should no effect on the input, since all input, including user-defined parameters
+are order-independent.
```

Comment:
not completely true btw.
Positions that retrieve other positions at initialization need to be ordered in the input file (same for other systems that don't have fancier machinery)
Physics will not add variables/ICs/time derivatives if they have already been defined (by another Physics or by the input, acting on the same tasks), so the ordering matters there too